src/components/pages/footer.jsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import './Footer.css'
+
+function Footer () {
+    return (
+        <footer className="footer"> 
+        <hr className="bar"/>
+        Made with Love for FOSS in India
+        </footer>
+    )
+}
+
+export default Footer;
```

Comment:
Add last line character

Reply:
What character am I missing here?

Reply:
![image](https://github.com/shravan20/fosstrak-india/assets/47430686/2fc3c5b6-f415-4364-9bad-d3213809a1cb)
Add a new line, similarly in other files where you see this.